mf-rdlab/common/services.json

```
@@ -0,0 +1,92 @@
+[
+   {
+      "img":"//demo.digital-enabler.eng.it/suite/static/imgs/icons/icon_",
```

```suggestion
      "img":"//rdlab.digital-enabler.eng.it/suite/static/imgs/icons/icon_",
```
and all the followings